packages/@react-spectrum/s2/src/Popover.tsx

```diff
@@ -229,11 +232,28 @@ export const PopoverBase = forwardRef(function PopoverBase(props: PopoverProps,
   );
 });
 
-export interface PopoverDialogProps extends Pick<PopoverProps, 'size' | 'hideArrow'| 'placement' | 'shouldFlip' | 'containerPadding' | 'offset' | 'crossOffset' | 'triggerRef' | 'isOpen' | 'onOpenChange'>, Omit<DialogProps, 'className' | 'style' | keyof GlobalDOMAttributes>, StyleProps {
-}
+type popoverOverrides = [
+  'overflow',
+  'padding',
+  'paddingStart',
+  'paddingEnd',
+  'paddingTop',
+  'paddingBottom',
+  'paddingX',
+  'paddingY',
+  'display',
+  'flexDirection',
+  'gap'
```

Comment:
if we allow overriding display and gap, then we need to allow these as well?
```
flexWrap,
columnGap,
rowGap,
alignItems,
alignContent,
justifyItems,
justifyContent,
gridTemplateColumns,
gridTemplateRows,
gridTemplateAreas,
gridAutoFlow,
gridAutoRows,
gridAutoColumns
```

Reply:
I guess? I was just going off the bare minimum we needed, and then perhaps we can see what users need, but I guess the above should also be included since they are directly related to display/gap